test/libsolidity/syntaxTests/viewPureChecker/gas_value_without_call.sol

```diff
@@ -0,0 +1,13 @@
+contract C {
+    function f() external payable {}
+	function g(address a) external pure {
+		a.call.value(42);
+		a.call.gas(42);
+		a.staticcall.gas(42);
```

Comment:
It's interesting that apparently we allow this for ``homestead``, as long as it's not called :-). Nothing wrong with that per se, since the only thing one can do with this expression is to directly call it anyways, but still surprising :-).